src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -4442,6 +4442,9 @@ string Inner()
 				}
 
 				throw new Exception("Unable to convert {0} for {1} with type {2}".InvariantCultureFormat(memberValue, memberName, propertyType));
+
+				static string? splitAndJoin(string? value)
```

Comment:
The debate is not yet completely settled, but local functions should use pascal casing. We'll need to add that to the coding guidelines.

```suggestion
				static string? SplitAndJoin(string? value)
```

Reply:
Will fix that.

I'd suggest enforcing coding-style via globalconfig or editorconfig.